ethereum/chain.py

```diff
@@ -263,21 +264,21 @@ def add_block(self, block):
                         break
                     b = self.get_parent(b)
                 replace_from = b.header.number
-                for i in xrange(replace_from, 2**63 - 1):
+                for i in range(replace_from, 2**63 - 1):
```

Comment:
Instead of range, I think `itertools.count` is a more appropriate tools for this.  It's inherently lazy so no need for py2/3 juggling and it does the intended *thing* which is to count upwards *indefinitely* until the break condition is met.

https://docs.python.org/2/library/itertools.html#itertools.count